### DISCUSSION TITLE
Save camera.doorbell Image to URL?

### DISCUSSION THREAD

**Discussion by Gtt1229** [fuatakgun/eufy_security#362] [Q&A]
Hello,

I would like to access the entity preview that is seen here:

![image](https://user-images.githubusercontent.com/23534272/173676323-e693c89b-da23-4904-a3cf-8e1371df81b3.png)

I am utilizing Notifications for FireTV to send a notification to the TV, and I want to attach an image URL in order to preview who is at the door.

Has anyone had luck doing this or a better way to accomplish this task?

**Comment by fuatakgun** [comment]
Have you set quality of stream and recording as low?
